client/post-editor/editor-more-options/copy-post.jsx

```diff
@@ -109,7 +109,6 @@ class EditorMoreOptionsCopyPost extends Component {
 					</Button>
 				</EditorDrawerLabel>
 				<Dialog
-					autoFocus={ false }
```

Comment:
For folks reviewing. Note this is unrelated to this PR but needed because it was causing a build failure: https://circleci.com/gh/Automattic/wp-calypso/195264

```
line 112, col 6, Error - The autoFocus prop should not be used, as it can reduce usability and accessibility for users. (jsx-a11y/no-autofocus)
```